tuxemon/db.py

```diff
@@ -725,18 +752,31 @@ def lookup(
     ) -> EnvironmentModel:
         pass
 
-    def lookup(self, slug: str, table: str = "monster") -> Mapping[str, Any]:
+    def lookup(
+        self, slug: str, table: str = "monster"
+    ) -> Union[
+        EconomyModel,
+        EncounterModel,
+        EnvironmentModel,
+        InventoryModel,
+        ItemModel,
+        Mapping[str, Any],
```

Comment:
Is that mapping REALLY needed? Also, can we use a base class instead of the full list?

Reply:
I don't think it makes sense to add a base type, as we don't use these models polymorphically.

I could make an alias for the union if you think it's too verbose.